node/cli/src/chain_spec.rs

```diff
@@ -187,6 +187,7 @@ pub fn get_account_id_from_seed(seed: &str) -> AccountId {
 	sr25519::Pair::from_string(&format!("//{}", seed), None)
 		.expect("static values are valid; qed")
 		.public()
+		.unchecked_into()
```

Comment:
`unchecked_into` is semantically unsafe, so shouldn't be in any production code. though it's kind of safe here, i'd prefer to use a real `.into()` since we are after all converting between two strong types.